framework/src/task_manager/task_manager_core.c

```diff
@@ -475,6 +476,28 @@ static int taskmgr_unicast_async(int handle, int caller_pid, void *data)
 	return OK;
 }
 
+int taskmgr_calc_time(struct timespec *time, int timeout)
```

Comment:
How about offset instead of timeout?
You want to know the time after some time (2nd argu). Isn't it?

And how about calcalated_time or result_time or abstime_of_offset instead of time?